DEPLOYMENT.md

```diff
@@ -1,4 +1,4 @@
-[![Version](https://img.shields.io/badge/version-1.0.0-beta-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
+[![Version](https://img.shields.io/badge/version-1.0.1--beta-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
```

Comment:
FYI @fabioberger you have to put two dashes here to make it work with shields.io.